Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -125,6 +125,9 @@ def __init__(self, session, tdef):
         self.finished_deferred_already_called = False
         self.is_bootstrap_download = False
 
+        # If the download is hidden in GUI
```

Comment:
It's not only hidden in the GUI, it is not returned when requested through the REST API (`GET /downloads`). Can you make the comment match this?